### ISSUE TITLE
TypeError: 'numpy.dtype' object is not callable

### ISSUE THREAD

**Issue by gautambak** [alteryx/featuretools#1356]
Hi There,

I have a question because I'm sure I'm doing something wrong here. I previously asked a question and resolved it but I cannot figure how it worked because it's not working anymore ( https://github.com/alteryx/featuretools/issues/1185 )

What I'm trying to do is I'm trying to take data from a panda dataframe and move it to Dask so I can pass it over to featuretools. It seems to be loading in task correctly but when I'm sending out the featuretools it was complaining that I had to set the variable types. So I did what I did last time which apparently worked but now it's not working and I'm getting a strange numpy error.

```
Here's my code:
import dask.dataframe as dd
dt = Main.dtypes.to_dict()

Main[catgoricalValues] = Main[catgoricalValues].astype('category')
Main[NumericColumns] = Main[NumericColumns].astype('float64')
dt.update(dict.fromkeys(catgoricalValues, ft.variable_types.Categorical))
dt.update(dict.fromkeys(NumericColumns, ft.variable_types.Numeric))

dask_df = dd.from_pandas(Main[catgoricalValues + NumericColumns ], npartitions=2

import featuretools as ft

# Make an entityset and add the entity
es = ft.EntitySet(id = 'Test')
# es = es.entity_from_dataframe(entity_id = 'data', dataframe = dask_df, make_index = False, index = 'index', variable_types = dt)
es = es.entity_from_dataframe(entity_id="dask_entity", dataframe=dask_df, index="index", variable_types=dt)
```

I get this error:
```
2 frames
/usr/local/lib/python3.7/dist-packages/featuretools/entityset/entityset.py in entity_from_dataframe(self, entity_id, dataframe, index, variable_types, make_index, time_index, secondary_time_index, already_sorted)
    521             secondary_time_index=secondary_time_index,
    522             already_sorted=already_sorted,
--> 523             make_index=make_index)
    524         self.entity_dict[entity.id] = entity
    525         self.reset_data_description()

/usr/local/lib/python3.7/dist-packages/featuretools/entityset/entity.py in __init__(self, id, df, entityset, variable_types, index, time_index, secondary_time_index, last_time_index, already_sorted, make_index, verbose)
     71 
     72         secondary_time_index = secondary_time_index or {}
---> 73         self._create_variables(variable_types, index, time_index, secondary_time_index)
     74 
     75         self.df = df[[v.id for v in self.variables]]

/usr/local/lib/python3.7/dist-packages/featuretools/entityset/entity.py in _create_variables(self, variable_types, index, time_index, secondary_time_index)
    255                 _v = vtype[0](v, self, **vtype[1])
    256             else:
--> 257                 _v = inferred_variable_types[v](v, self)
    258             variables += [_v]
    259         # convert data once we've inferred

TypeError: 'numpy.dtype' object is not callable
```

**Comment by thehomebrewnerd** [comment]
@gautambak I am not sure, but I think what could be happening here is that the variable types dictionary you are defining might contain some variable types that are specified incorrectly.

When creating an Entity from a Dask dataframe, every column in the dataframe must have a corresponding Featuretools variable type defined for it. When you run `dt = Main.dtypes.to_dict()` you are creating a dictionary that contains a key for every column in the dataframe, and then you update some of those keys with featuretools variable types for categorical and numeric columns. 

Does your dataframe contain any additional columns that are not categorical or numeric? If so, you also need to make sure to add specify variable types for those columns. Otherwise, the default dtype for those columns could carry over from your call to `Main.dtypes.to_dict()` and that could generate the error you are seeing.

So, to summarize, please confirm that your `dt` dict has a key for every column in your dataframe, and that all of the values are valid Featuretools variable types (should be a subclass of `featuretools.Variable`).

If you are still having issues after this, let us know and we can investigate further.

**Comment by gautambak** [comment]
Thank you @thehomebrewnerd  that worked. It was a dumb bug that I was introducing by adding all items then later specifically overwriting with FT column types. This solved it! Thank you!

**Comment by thehomebrewnerd** [comment]
@gautambak You're welcome. Glad that helped resolve your issue!

**Comment by gautambak** [comment]
@thehomebrewnerd  Thanks. you helped me with a previous issue as well! You rock Nate.

I have another issue I just raised which might be my biggest one yet :-)
